pkg/oauthserver/setup_test.go

```diff
@@ -0,0 +1,119 @@
+package oauthserver_test
+
+import (
+	"net/http"
+
+	oauthv1 "github.com/openshift/api/oauth/v1"
+	osinv1 "github.com/openshift/api/osin/v1"
+	fakeoauthclient "github.com/openshift/client-go/oauth/clientset/versioned/fake"
+	typedv1 "github.com/openshift/client-go/oauth/clientset/versioned/typed/oauth/v1"
+	bootstrap "github.com/openshift/library-go/pkg/authentication/bootstrapauthenticator"
+	"github.com/openshift/oauth-server/pkg/config"
+	"github.com/openshift/oauth-server/pkg/oauthserver"
+	"github.com/openshift/oauth-server/pkg/userregistry/identitymapper"
+	"golang.org/x/crypto/bcrypt"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	fakekube "k8s.io/client-go/kubernetes/fake"
```

Comment:
please sort the packages:
1. native go imports
2. 3rd party vendor import
3. k8s imports
4. openshift imports
5. project internal imports

Reply:
oki doki